doc/source/templates/04_finetuning_llms_with_deepspeed/finetune_hf_llm.py

```diff
@@ -122,6 +123,16 @@ def evaluate(
         perplexity = float("inf")
     return perplexity, eval_loss
 
+def _test_tokenizer(model_name):
```

Comment:
Maybe also leave a breadcrumb here for https://github.com/huggingface/transformers/issues/25176 ? Otherwise context for this test will be lost over time